velox/dwio/parquet/tests/ParquetWriterTestBase.h

```diff
@@ -35,17 +36,23 @@ class ParquetWriterTestBase : public testing::Test {
     rootPool_ = memory::defaultMemoryManager().addRootPool("ParquetWriterTest");
     leafPool_ = rootPool_->addLeafChild("SinkTest");
     tempPath_ = exec::test::TempDirectoryPath::create();
+    rowType_ =
+        ROW({"c0", "c1", "c2", "c3", "c4", "c5"},
+            {BIGINT(), INTEGER(), SMALLINT(), REAL(), DOUBLE(), VARCHAR()});
   }
 
   std::vector<RowVectorPtr> createBatches(
-      const std::shared_ptr<const Type>& rowType,
       uint64_t batchNum,
-      uint64_t capacity) {
+      uint64_t vectorSize) {
+    VectorFuzzer::Options opts;
```

Comment:
Notice that this logic will create same exact batch every time. Is this the intention?

Reply:
I wanted to use it to generate some data to write no matter what the content is.